docs/rbac/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _rbac-index:
+
+Role-based Access Control
+=========================
+
+Role-based access control (RBAC) **restricts** access to content based on a user's role within an
+organization. The role consists of one or more permissions. Users having a proper set of roles can
+view, modify, or delete content hosted on different endpoints.
+
+By default, all container repositories are accessible by anyone, unless the opposite is *explicitly*
```

Comment:
I think you are referring to registry pull access here. Maybe something like `... repositories content is accessible via podman pull by anyone...`